### PR TITLE
Remove code paths for python <3.11

### PR DESCRIPTION
**Pull request by JHolba**
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Make sure unit tests pass locally after every commit (`git rebase -i main
      --exec 'pytest tests/ert/unit_tests -n logical -m "not integration_test"'`)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: src/ert/config/parsing/hook_runtime.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from ert.enum_shim import StrEnum
```
Comment (eivindjahren) [inline src/ert/config/parsing/hook_runtime.py:1]: Remember to delete src/ert/enum_shim.py

**Review by eivindjahren — APPROVED** [review]
:scissors:

File: tests/everest/entry_points/test_visualization_entry.py [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-from everest.bin.visualization_script import visualization_entry
-from everest.detached import ServerStatus
-from tests.everest.utils import capture_streams
-
-
-@patch(
-    "everest.bin.visualization_script.everserver_status",
-    return_value={"status": ServerStatus.completed},
-)
-@pytest.mark.skipif(sys.version_info.major < 3, reason="requires python3 or higher")
```
Comment (eivindjahren) [inline tests/everest/entry_points/test_visualization_entry.py:16]: This seems to be a test that is skipped for python < 3 which means its currently running for py 3.11

**Review by eivindjahren — CHANGES_REQUESTED** [review]
Cut out a bit too much.

**Comment by JHolba** [comment]
> Cut out a bit too much.

Urgh. Brainfart. Will fix

File: tests/everest/entry_points/test_visualization_entry.py (hunk 2 above)
Comment (eivindjahren) [reply tests/everest/entry_points/test_visualization_entry.py:16]: Should just remove the skip that only applies to python 2 which is out of scope.

Comment (JHolba) [reply tests/everest/entry_points/test_visualization_entry.py:16]: fixed

**Review by eivindjahren — APPROVED** [review]
:scissors:

**Pull request merged**